.release-notes/81.md

```diff
@@ -0,0 +1,62 @@
+## Modify finished API remove assumption all responses are one-shot
+
+NOTE: This _*is*_ a breaking change.
+
+The existing Handler API made the assumption that all
```

Comment:
if the API is existing then it would `make` assumptions. There's a tense disagreement here.